### ISSUE TITLE
data from SDM120M incomplete

### ISSUE THREAD

**Issue by sicheste** [volkszaehler/mbmd#340]
If I add the SDM120M as `SDM220` to the config, I will not get all values. Especially the `Power (W)` is missing. If I add it as a `SDM` I get this value.

As `SDM220`:
![image](https://github.com/volkszaehler/mbmd/assets/7395079/3391c8fa-1e41-4eab-9201-a64df21ed4ff)

As `SDM`:
![image](https://github.com/volkszaehler/mbmd/assets/7395079/b593eaab-736c-420e-896f-e345b69b7e79)

Btw: What is the `name` field in the config for? The name (here `SDM2201.3` and `SDM1.3` is constructed by type + position in the alphabetically sorted list of sensors + `.` or `-` + bus id. Thus the names in the API and MQTT are unstable.

**Comment by sicheste** [comment]
I just recognized that the current docker image is already 4 years old. I will try in the next days to build my own one using the current master branch and give it a try. Maybe it is time for a new version :)

**Comment by sicheste** [comment]
So, I had some time. I have created a new docker image locally, based on the current master branch. Running `mbmd scan` was not detecting the SDM120M. But manually giving it as SDM220:3 (it is number 3 on the bus) it works. Data in MQTT and on the web interface are fine.

So, still open:
* The detection is not working (at least here).
* The names are constructed using IDs, which are based on the alphabetical order of the meters. This makes the naming unstable when adding / removing meters.

**Comment by isarrider** [comment]
https://hub.docker.com/r/isarrider/mbmd/tags
try mine...

**Comment by isarrider** [comment]
nah, I just wanted to give @sicheste and others a docker image that isnt old...
(actually thats why I started also building mine, bc there is no newer)

**Comment by andig** [comment]
Thank you. Would be happy for build workflow PR- don't have the time...

**Comment by isarrider** [comment]
I know the feeling but I am always hoping for the young IT Students or similar...
